python/semantic_kernel/connectors/memory/astradb/astra_client.py

```diff
@@ -1,18 +1,17 @@
 import json
-from importlib.metadata import PackageNotFoundError, version
 from typing import Dict, List, Optional
 
 import aiohttp
 
 from semantic_kernel.connectors.memory.astradb.utils import AsyncSession
+from semantic_kernel.connectors.telemetry import APP_INFO
 from semantic_kernel.exceptions import ServiceResponseException
 
-SEMANTIC_KERNEL_VERSION: Optional[str]
-try:
-    SEMANTIC_KERNEL_VERSION = version("semantic_kernel")
+ASTRA_CALLER_IDENTITY: str
+SEMANTIC_KERNEL_VERSION = APP_INFO.get("Semantic-Kernel-Version")
+if SEMANTIC_KERNEL_VERSION:
     ASTRA_CALLER_IDENTITY = f"semantic_kernel/{SEMANTIC_KERNEL_VERSION}"
```

Comment:
shouldn't this also be semantic-kernel instead of with an underscore?

Reply:
I would have a slight preference for an underscore as so far we tried to stick to the very Python package name convention. But if this conflicts with e.g. branding guidelines for SK, happy to comply! Just let me know, and thank you for the super fast response :)

Reply:
I get where you are coming from, but the package is called semantic-kernel, so ultimately less confusing if someone wants to check the package that a certain set of calls are coming from!

Reply:
All right, it makes sense. As I said, no problem - I updated the PR accordingly. Thank you!